apps/web/components/ui/VsfLogo.vue

```diff
@@ -1,13 +1,21 @@
 <template>
   <picture>
     <template v-if="imageExtension === 'svg'">
-      <NuxtImg :src="imagePath" alt="Logo" class="w-100 h-10 py-2" width="150" height="40" preload />
+      <NuxtImg
+        :src="imagePath"
+        :alt="`${{ storeName }} logo`"
```

Comment:
I think you should use the binding syntax here: 

`        :alt="`${storeName} logo`"`

Reply:
This syntax throws a parsing error for me.